Dockerfile.service_discovery.multi

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+ARG RUST_VERSION=1.70
+
+FROM --platform=$BUILDPLATFORM docker.io/library/rust:${RUST_VERSION} AS builder
```

Comment:
Could we add a line or two to the README for the usage of this Dockerfile? I don't think we have any container documentation in this repository yet, but at least if we could show the possible ways to build this one I think it would be a good start